include/Knob.h

```diff
@@ -60,6 +62,12 @@ class LMMS_EXPORT Knob : public QWidget, public FloatModelView
 	Q_PROPERTY(QColor outerColor READ outerColor WRITE setOuterColor)
 	Q_PROPERTY(QColor lineColor READ lineColor WRITE setlineColor)
 	Q_PROPERTY(QColor arcColor READ arcColor WRITE setarcColor)
+
+	Q_PROPERTY(QColor lineActiveColor MEMBER m_lineActiveColor)
+	Q_PROPERTY(QColor lineInactiveColor MEMBER m_lineInactiveColor)
+	Q_PROPERTY(QColor arcActiveColor MEMBER m_arcActiveColor)
+	Q_PROPERTY(QColor arcInactiveColor MEMBER m_arcInactiveColor)
```

Comment:
One thing that might be cool to do and allow even cooler themes is to allow gradients instead of single colors. Changing these to `QBrush` would allow you to define gradients or colors.

Reply:
Thanks @Veratil  for the suggestion. I replaced QColor by QBrush, below is an example. You get the idea from the picture, but a `QLinearGradient` does not result in the intended behavior here, see e.g. the red color at low volumes (I think this is due to the gradient being applied independent of the length of the active line). Furthermore, you can observe that `m_lineColor` is used both for the "middle line" and the active part of the "arc line". 
Maybe we could address this feature separately?

![qbrush_knob](https://user-images.githubusercontent.com/64359888/86179566-ce287580-bb2a-11ea-9f70-632b493a071f.gif)

Reply:
You wouldn't use a linear gradient in that instance, you'd use a conical gradient in the CSS. Example from Qt documentation:
```css
/* conical gradient from white to green */
QTextEdit {
    background: qconicalgradient(cx:0.5, cy:0.5, angle:30,
                stop:0 white, stop:1 #00FF00)
}
```

At the same time though, if you wanted single color you can define the brush to be the normal `rgba(...)`.